### PR TITLE
Switch Drupal 9 clean to `create-project` workflow.

### LINKED ISSUE #559: Use composer for Drupal 9 installs
https://github.com/civicrm/civicrm-buildkit/blob/3fbb12b7e9fb97b0c03162c136e2e7051f8248a7/app/config/drupal9-clean/download.sh#L11

We still appear to be using `drush8` to download Drupal 9 instead of using Composer. This throws several warnings:

```
Drush\UpdateService\Project::parseXml(): Node no longer exists Project.php:232                                                            [warning]
There are no stable releases for project drupal.                                                                                                                  [warning]
Drupal  has reached EOL, so there are no stable releases for any contrib projects. Selected the best release, drupal      [warning]
9.0.5.
``` 

We should be using composer based install/project workflows for all Drupal 8/9 installs in Buildkit.

### PR DESCRIPTION
**Pull request by MikeyMJCO**
Fixes #559

### CODE REVIEW

File: app/config/drupal9-clean/download.sh [hunk 1]
```diff
@@ -1,19 +1,19 @@
+
```
Comment (seamuslee001) [inline app/config/drupal9-clean/download.sh:1]: @MikeyMJCO any reason for this?

**Review by seamuslee001 — APPROVED** [review]
This looks fine just a bit odd with the first line

**Comment by MikeyMJCO** [comment]
So this has one issue remaining:

`Failed to locate valid civi root: /opt/buildkit/build/drupal9-clean/web/vendor/civicrm/civicrm-core`

**Comment by MikeyMJCO** [comment]
Running test builds.

**Comment by MikeyMJCO** [comment]
Tested and working.

**Pull request merged**